.gitignore

```diff
@@ -1,5 +1,6 @@
 # dependencies
 /node_modules
+package-lock.json
```

Comment:
I see gitignore as containing files that, under normal usage, could be on disk but we would want to ignore.

package-lock.json is not something users should have under normal usage, and is a sign that something has gone wrong (i.e. they've used npm instead of yarn)

I would prefer this was not added to gitignore

Reply:
thanks I understand. This has been reverted back too